benchmarks/routines/gemm.py

```
@@ -836,6 +842,235 @@ def run_backend(backend, input_fp8, mat2_fp8, input_inv_s, mat2_inv_s):
+        )
+
+    if args.batch_size != 1:
+        raise ValueError("mm_fp8 is a 2D GEMM; only --batch_size 1 is supported.")
+    if k % 128 != 0:
+        raise ValueError(f"mm_fp8 requires k % 128 == 0, got k = {k}")
+    if n % 32 != 0:
+        raise ValueError(f"mm_fp8 requires n % 32 == 0, got n = {n}")
+    ## Done parsing input arguments
+
+    ## Prepare input tensors
+    input = torch.randn([m, k], device=device, dtype=torch.bfloat16)
```

Ruff (0.15.20)

[error] Variable `input` is shadowing a Python builtin

(A001)